src/viking.js

```diff
@@ -1,17 +1,65 @@
 // Soldier
-class Soldier {}
+class Soldier {
+  constructor(health, strength) {
+    this.health = health;
+    this.strength = strength;
+
+  }
+  attack() {
+    return this.strength;
+  }
+  receiveDamage(damage) {
+    this.health -= damage;
+  }
+}
 
 // Viking
-class Viking {}
+class Viking extends Soldier {
+  constructor(name, health, strength) {
+    super(name, health, strength);
+    this.name = name;
```

Comment:
Hi Alex, insgesamt gute Arbeit! :) 
Hier eine Sache, in super muss immer das rein, was du von der Eltern Klasse (Soldier) übernimmst, also nur health und strength und in den constructor müssen dann alle Eigenschaften gelistet werden (hast du ja auch). In der Saxon Class hast du das richtig gemacht, also so:
```
class Viking extends Soldier {
  constructor(name, health, strength) {
    super(health, strength);
    this.name = name;
  } 
```